### PR TITLE
Babanews

### PR DESCRIPTION
**Pull request by antkiewiczk**

### CODE REVIEW

File: apps/babanews/pages/index.tsx [hunk 1]
```diff
@@ -31,7 +33,7 @@ const Frontpage = styled(PageContainer)`
 `
 
 export default function Index() {
-  return <Frontpage slug={''} />
+  return <Frontpage slug={'home'} />
```
Comment (Itrulia) [inline apps/babanews/pages/index.tsx:36]: We should use a page without a slug ideally.

File: apps/babanews/src/components/babanews/fullwidth-banner/fullwidth-banner.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const Banner = styled('div')`
+  aspect-ratio: 2.7/1;
+  width: 100%;
+  background-color: ${({theme}) => theme.palette.info.main};
+`
+
+const imageStyles = (theme: Theme) => css`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
```
Comment (Itrulia) [inline apps/babanews/src/components/babanews/fullwidth-banner/fullwidth-banner.tsx:15]: Check this on safari/iOS might cause issues. We have the same bug for our teasers rn

File: apps/babanews/src/components/website-builder-overwrites/blocks/teaser-overwrite.tsx
Comment (Itrulia) [inline apps/babanews/src/components/website-builder-overwrites/blocks/teaser-overwrite.tsx:1]: dont think we want these

File: apps/babanews/src/components/babanews/fullwidth-banner/fullwidth-banner.tsx (hunk 2 above)
Comment (antkiewiczk) [reply apps/babanews/src/components/babanews/fullwidth-banner/fullwidth-banner.tsx:15]: checked on safari, no issues. I don't have an ios device

File: apps/babanews/pages/index.tsx (hunk 1 above)
Comment (antkiewiczk) [reply apps/babanews/pages/index.tsx]: removed

File: apps/babanews/pages/index.tsx [hunk 3]
```diff
@@ -10,7 +10,10 @@ import {GetStaticProps} from 'next'
 import getConfig from 'next/config'
 
 const Frontpage = styled(PageContainer)`
-  ${TeaserGridFlexBlockWrapper}:first-of-type ${TeaserWrapper}:first-of-type {
+  gap: ${({theme}) => theme.spacing(6)};
+  padding-bottom: ${({theme}) => theme.spacing(6)};
+  ${TeaserGridFlexBlockWrapper}:first-of-type
```
Comment (Itrulia) [inline apps/babanews/pages/index.tsx:15]:
```suggestion

  ${TeaserGridFlexBlockWrapper}:first-of-type
```

File: apps/babanews/src/components/layout/full-width-container.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import {styled} from '@mui/material'
+
+export const FullWidthContainer = styled('main')<{backgroundColor?: string}>`
+  background-color: ${({theme, backgroundColor}) => backgroundColor || theme.palette.info.main};
+  grid-column: 1 / -1;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
```
Comment (Itrulia) [inline apps/babanews/src/components/layout/full-width-container.tsx:7]: Dont think this is needed

File: apps/babanews/src/components/website-builder-overwrites/blocks/list-teaser.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  object-fit: cover;
+  grid-column: 1/7;
+  aspect-ratio: 3/2;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    aspect-ratio: 16/9;
+  }
+`
+
+export const fluidTypography = (minSize: number, maxSize: number): string => {
```
Comment (Itrulia) [inline apps/babanews/src/components/website-builder-overwrites/blocks/list-teaser.tsx:22]: Do we use this anywhere?

File: apps/babanews/pages/_app.tsx [hunk 6]
```diff
@@ -168,26 +116,17 @@ function CustomApp({Component, pageProps}: CustomAppProps) {
               <meta name="msapplication-TileColor" content="#ffffff" />
               <meta name="theme-color" content="#ffffff" />
             </Head>
-
-            <Spacer>
+            <div className={hankenGrotesk.className}>
```
Comment (Itrulia) [inline apps/babanews/pages/_app.tsx:119]: Removal of the spacer now has the problem that if the page body isn't big enough, the footer isn't at the bottom of the page FYI

File: apps/babanews/src/styles/theme.ts [hunk 7]
```diff
@@ -0,0 +1,47 @@
+import {createTheme} from '@mui/material'
+
+const {
+  palette: {augmentColor}
+} = createTheme()
+
+const theme = createTheme({
+  typography: {
```
Comment (Itrulia) [inline apps/babanews/src/styles/theme.ts:8]: Prob should be removed as it comes from the parent theme

File: apps/babanews/src/styles/theme.ts [hunk 8]
```diff
@@ -0,0 +1,47 @@
+import {createTheme} from '@mui/material'
+
+const {
+  palette: {augmentColor}
+} = createTheme()
+
+const theme = createTheme({
```
Comment (Itrulia) [inline apps/babanews/src/styles/theme.ts:7]: We should inherit from the WePtheme

```suggestion
import {theme as WePTheme} from "@wepublish/ui"

const theme = createTheme(WePTheme, {
```

File: apps/babanews/src/styles/theme.ts [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+          fontWeight: 'bold',
+          ':hover': {
+            borderWidth: '3px'
+          }
+        }
+      }
+    }
+  }
+})
+export default theme
```
Comment (Itrulia) [inline apps/babanews/src/styles/theme.ts:47]:
```suggestion

export default theme
```

File: apps/bajour/src/components/website-builder-overwrites/blocks/single-teaser.tsx [hunk 10]
```diff
@@ -74,13 +74,13 @@ export const SingleTeaser = styled(TeaserOverwrite)`
     }
   }
 
-  ${({theme}) => theme.breakpoints.up('xl')} {
+  /* ${({theme}) => theme.breakpoints.up('xl')} {
```
Comment (Itrulia) [inline apps/bajour/src/components/website-builder-overwrites/blocks/single-teaser.tsx:77]: Should be removed and not commented out i guess

File: apps/babanews/src/components/website-builder-overwrites/blocks/teaser.tsx [hunk 11]
```diff
@@ -0,0 +1,11 @@
+import {BuilderTeaserProps, Teaser} from '@wepublish/website'
+
+import {ListTeaser} from './list-teaser'
+
+export const BabanewsTeaser = (props: BuilderTeaserProps) => {
+  if (!props.numColumns) {
```
Comment (Itrulia) [inline apps/babanews/src/components/website-builder-overwrites/blocks/teaser.tsx:6]: Is this really what we want? Shouldn't this be solved in the block renderer?

File: apps/babanews/pages/_app.tsx (hunk 6 above)
Comment (antkiewiczk) [reply apps/babanews/pages/_app.tsx:123]: this should be solved now as well, as I added spacer but in different place

File: apps/babanews/src/components/website-builder-overwrites/blocks/teaser.tsx (hunk 11 above)
Comment (antkiewiczk) [reply apps/babanews/src/components/website-builder-overwrites/blocks/teaser.tsx]: that's correct. I've removed it and moved the logic to block-renderer

**Review by Itrulia — APPROVED** [review]

**Review by Itrulia — APPROVED** [review]

**Pull request merged**